scripts/us_eia/opendata/process/ng.py

```diff
@@ -0,0 +1,66 @@
+"""EIA Natural Gas Dataset specific functions."""
+
+import re
+
+# List of series in Pattern #1 that include non-US countries. Unfotunately this
```

Comment:
```suggestion
# List of series in Pattern #1 that include non-US countries. Unfortunately this
```

Reply:
also, could you describe the significance of the N/S prefix?

Reply:
Done.